appinfo/info.xml

```diff
@@ -5,7 +5,7 @@
 	<summary>Integration of Zammad user support/ticketing solution</summary>
 	<description><![CDATA[Zammad integration provides a dashboard widget displaying your important notifications,
 	a search provider for tickets and notifications for new open tickets.]]></description>
-	<version>2.0.6</version>
+	<version>2.0.7</version>
```

Comment:
Generally, I'd recommend bumping the minor version for changes in supported nextcloud version, but it's up to you :)